context/http.go

```diff
@@ -62,6 +62,16 @@ func RemoteIP(r *http.Request) string {
 	return addr
 }
 
+// getXRequestID get X-Request-ID from HTTP header
+// or generate an UUID if not found in header
+func getXRequestID(r *http.Request) string {
+	rid := r.Header.Get("X-Request-ID")
```

Comment:
Do you have a source for the specification of X-Request-ID field? Is it defined in the OCI spec?